config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
+  get    '/signup',  to: 'users#new'
+
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   resources :puppies, only: [:index]
+  resources :users
```

Comment:
Can you add an ``only: [:new, :create, :edit, :update, :destroy]`` to this to prevent unnecessary actions?